### PR TITLE
fix(wc): Pie/Donut chart stop working past 7 values

### PR DESCRIPTION
**Pull request by code2nguyen**
- issue #785

### CODE REVIEW

File: libs/web-components/charts/src/pie-chart.ts [hunk 1]
```diff
@@ -30,8 +30,8 @@ export class PieChart extends ApexChartsWrapper {
         return this._data;
     }
     public set data(value: number[]) {
-        this._data = value.sort((a, b) => b - a);
```
Comment (bcldvd) [inline libs/web-components/charts/src/pie-chart.ts:33]: Sorting is lost. Need to have it

Comment (code2nguyen) [reply libs/web-components/charts/src/pie-chart.ts:33]: it create an error, because data and label need to be sync.

Comment (bcldvd) [reply libs/web-components/charts/src/pie-chart.ts:33]: Can we have the data sorted some other ways ? :)

Comment (code2nguyen) [reply libs/web-components/charts/src/pie-chart.ts:33]: This not link with this issue. Please create an issue about it if you need. Please considering about following points:

- The sync between data and label is more expensive some time can create errors if user change data -> sort data -> sort label ... then user update labels but list of labels does not sync back from chart component.
- Dont know which order (desc/asc) is better, it depends de lire direction (rtl or ltr) or event depends on order of theme colors.
-User normalement, can easily transform data outside the component. Component will display as its input. It make component more predicable.

**Review by dorsaffrigui — APPROVED** [review]

**Pull request merged**